lib/game_setting/setting_types.rb

```diff
@@ -0,0 +1,9 @@
+module SettingTypes
```

Comment:
Why do you have the `SettingTypes` module?

Reply:
I have a few places in my app that relies on the string types. So I stored it in one place instead of hard coding strings in 2 places (tic_tac_toe)